packages/evals/helpers.py

```diff
@@ -7,16 +9,17 @@
 import logging
 import os
 from pathlib import Path
-from typing import List, TypedDict
+from typing import List
 
 import mlflow
 from dotenv import load_dotenv
 from mlflow.openai import autolog
+from typing_extensions import TypedDict
```

Comment:
@heyitsaamir I said I would tag you on this change just in case :)

Reply:
Huh, weird, g2k!